packages/os/healthdog.timer

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Scheduled Healthdog Pings
+
+[Timer]
+# Don't run missed executions
+Persistent=false
+# Run 120 seconds after boot for the first time
+OnBootSec=120
+# Run every 6 hours thereafter
+OnUnitActiveSec=21600
```

Comment:
```suggestion
OnUnitActiveSec=21600
# Don't fire at exactly the same second across machines started together.
RandomizedDelaySec=10
# We don't want to extend the startup report too long after the requested time.
AccuracySec=10
```

Reply:
Addressed in [f18d841](https://github.com/bottlerocket-os/bottlerocket/compare/ec0abbf943fe3806294f53fd4e118be3d1f529fc..f18d841ee1c2b35d5f09136ea3f7136e2a4d65ae)